src/current/_includes/v23.1/cdc/lagging-ranges.md

```diff
@@ -1,10 +1,12 @@
-{% include_cached new-in.html version="v23.1.12" %} Use the `changefeed.lagging_ranges` metric to track the number of ranges that are behind in a changefeed. This is calculated based on the [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}):
+{% include_cached new-in.html version="v23.1.12" %} Use the `changefeed.lagging_ranges` metric to track the number of [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) that are behind in a changefeed. This is calculated based on the [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}):
 
 - `changefeed.lagging_ranges_threshold` sets a duration from the present that determines the length of time a range is considered to be lagging behind, which will then track in the [`lagging_ranges`]({% link {{ page.version.version }}/monitor-and-debug-changefeeds.md %}#using-changefeed-metrics-labels) metric. Note that ranges undergoing an [initial scan]({% link {{ page.version.version }}/create-changefeed.md %}#initial-scan) for longer than the threshold duration are considered to be lagging. Starting a changefeed with an initial scan on a large table will likely increment the metric for each range in the table. As ranges complete the initial scan, the number of ranges lagging behind will decrease.
     - **Default:** `3m`
 - `changefeed.lagging_ranges_polling_interval` sets the interval rate for when lagging ranges are checked and the `lagging_ranges` metric is updated. Polling adds latency to the `lagging_ranges` metric being updated. For example, if a range falls behind by 3 minutes, the metric may not update until an additional minute afterward.
     - **Default:** `1m`
 
+{% include_cached new-in.html version="v23.1.29" %} Use the `changefeed.total_ranges` metric to monitor the number of ranges that are watched by [nodes participating in the changefeed job]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}). If you're experiencing lagging ranges, `changefeed.total_ranges` may indicate that the number of ranges watched by nodes in the job is unbalanced. You may want to try [pausing]({% link {{ page.version.version }}/pause-job.md %}) the changefeed and then [resuming]({% link {{ page.version.version }}/resume-job.md %}) it, so that the changefeed replans the work in the cluster. `changefeed.total_ranges` shares the same polling interval as the `changefeed.lagging_ranges` metric, which is controlled by the `changefeed.lagging_ranges_polling_interval` cluster setting.
```

Comment:
@andyyang890 Does this explanation make sense re "... the number of ranges that are watched by nodes participating in the changefeed job..."?

Reply:
Could we say something like "the number of ranges that are watched by aggregator processors participating in the changefeed job"? The reason I want to call out that it's aggregators specifically is because it doesn't count ranges for the frontier processor. I see that we use the "aggregator processor" terminology on the linked page so I don't think it would be jargony.

Reply:
OK. Yes, you saw what I was attempting to avoid, but makes sense to be specific here. I'll update.

Reply:
@andyyang890 PTAL! Thanks